test/e2e/play_kube_test.go

```diff
@@ -209,6 +209,16 @@ var _ = Describe("Podman generate kube", func() {
 		processTestResult(f)
 	})
 
+	It("podman play kube fail with nonexist authfile", func() {
```

Comment:
@TomSweeneyRedHat these tests for play_kube_test.go

Reply:
kk, thx.